pkg/sql/set_zone_config.go

```diff
@@ -123,7 +124,7 @@ func (p *planner) SetZoneConfig(ctx context.Context, n *tree.SetZoneConfig) (pla
 		// We have a CONFIGURE ZONE = <expr> assignment.
 		// This can be either a literal NULL (deletion), or a string containing YAML.
 		// We also support byte arrays for backward compatibility with
-		// previous versions of CockroachDB.
+		// previous versions of CockrofachDB.
```

Comment:
Minor nit `s/CockrofachDB/CockroachDB/g`

Reply:
Thank you @odeke-em! I missed that, will fix.